CodeStarterCamp_Week2/step2!!.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Step2.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 이지원 on 2023/05/23.
+//
+
+import Foundation
+
+
+func getLottoNumber() -> Set<Int> {
+    
+    var lottoNumbers = Set<Int>()
+    
+    while lottoNumbers.count < 6 {
+        lottoNumbers.insert(Int.random(in: 1...45))
+    }
+    
+    return lottoNumbers
+}
+
+func compareLottoResult(to lottos: Set<Int>, from myLottos: [Int]) {
+    let result = Array(lottos.intersection(myLottos)).sorted()
```

Comment:
```suggestion
    let result = lottos.intersection(myLottos).sorted()
```

`Set` 타입 또한 `sorted()` 메서드가 있기 때문에 굳이 `Array`로 타입을 바꾼 다음에 호출할 필요는 없을 것 같습니다.